idea/common/src/jsonrpc-errors.ts

```diff
@@ -99,6 +99,16 @@ export const JSONRPC_ERRORS = {
     code: -32403,
     message: 'Forbidden',
   },
+  InvalidAddress: {
+    name: 'InvalidAddress',
+    code: -32400,
+    message: 'Address is invalid',
+  },
+  NotSupportedAddress: {
+    name: 'NotSupportedAddress',
+    code: -32400,
```

Comment:
maybe it's better have different codes for different errors?

Reply:
Yes, gonna do it in a different pr